### PR TITLE
Adding ko-fi floating button. Some styling fixes needed but it should…

### PR DESCRIPTION
**Pull request by afuggetta**
… allow to get donations now

### CODE REVIEW

**Review by kruzaavn — APPROVED** [review]

File: ui/app/index.html [hunk 1]
```diff
@@ -9,5 +9,14 @@
...
     <script type="module" src="/src/main.js"></script>
+    <script src='https://storage.ko-fi.com/cdn/scripts/overlay-widget.js'></script>
+    <script>
+      kofiWidgetOverlay.draw('jtf191', {
+        'type': 'floating-chat',
+        'floating-chat.donateButton.text': 'Donate',
+        'floating-chat.donateButton.background-color': '#00b9fe',
+        'floating-chat.donateButton.text-color': '#fff'
+      });
+    </script>
```
Comment (kruzaavn) [inline ui/app/index.html:20]: This has got to go direct into the base html?

Comment (afuggetta) [reply ui/app/index.html:20]: not necessarily. Let me see if I can move the script

Comment (afuggetta) [reply ui/app/index.html]: @kruzaavn I moved it to the component. Looks a bit cleaner.
Still needs some styling, but if you're good with it we can deploy so people can start donating

**Review by kruzaavn — APPROVED** [review]

**Comment by kruzaavn** [comment]
im good hit the button

**Comment by afuggetta** [comment]
🚢

**Pull request merged**